### PR TITLE
fix: gst profile env clean up and yolov5s_efficientnet no hardcoded var

### PR DESCRIPTION
**Pull request by vli11**
yolov5-cpu.env and yolov5-gpu.env clean up and yolov5s_efficientnet use env variable values

Closes: #410

## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #410 

## Anything the reviewer should know when reviewing this PR?
tested with integrated gpu
tested with cpu (core)
tested with serpent canyon dgpu
tested yolov5s.sh, yolov5s_full.sh, yolov5s_effnetb0.sh with existing env files with platform core, dgpu, dgpu.1

## Test Instructions if applicable
1. edit configs/opencv-ovms/cmd_client/res/gst/configuration.yaml, update PipelineInputArgs: "--pipeline_script_choice yolov5s_effnetb0.sh"
2. start the pipeline with cmd: PIPELINE_PROFILE="gst" RENDER_MODE=0 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0

### CODE REVIEW

File: configs/opencv-ovms/gst/framework-pipelines/yolov5_pipeline/yolov5s_effnetb0.sh [hunk 1]
```diff
@@ -11,11 +11,11 @@ if [ "$INPUTSRC_TYPE" == "REALSENSE" ]; then
...
\ No newline at end of file
+gst-launch-1.0 $inputsrc ! $DECODE ! gvadetect model-instance-id=odmodel name=detection model=/home/pipeline-server/models/yolov5s/FP16-INT8/1/yolov5s.xml model-proc=/home/pipeline-server/models/yolov5s/FP16-INT8/1/yolov5s.json threshold=.5 device=$DEVICE $PRE_PROCESS $DETECTION_OPTIONS ! gvatrack name=tracking tracking-type=zero-term-imageless ! queue max-size-bytes=0 max-size-buffers=0 max-size-time=0 ! gvaclassify model-instance-id=clasifier labels=/home/pipeline-server/models/efficientnet-b0/1/imagenet_2012.txt model=/home/pipeline-server/models/efficientnet-b0/FP32-INT8/1/efficientnet-b0.xml model-proc=/home/pipeline-server/models/efficientnet-b0/efficientnet-b0.json device=$DEVICE inference-region=roi-list name=classification $PRE_PROCESS $CLASSIFICATION_OPTIONS ! gvametaconvert name=metaconvert add-empty-results=true ! gvametapublish name=destination file-format=2 file-path=/tmp/results/r$cid_count.jsonl ! fpsdisplaysink video-sink=fakesink sync=true --verbose 2>&1 | tee >/tmp/results/gst_launch$cid_count.log >(stdbuf -oL sed -n -e 's/^.*current: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline$cid_count.log)
```
Comment (jim-wang-intel) [inline configs/opencv-ovms/gst/framework-pipelines/yolov5_pipeline/yolov5s_effnetb0.sh:21]: could you also make it consistent with other two having bash_cmd and echo it out?

**Review by jim-wang-intel — COMMENTED** [review]
Do we have any changes to env files? if so, please check them in.

**Comment by vli11** [comment]
> Do we have any changes to env files? if so, please check them in.

yes, please see 'tested' for reviewer to know.

**Comment by jim-wang-intel** [comment]
> > Do we have any changes to env files? if so, please check them in.
> 
> yes, please see 'tested' for reviewer to know.

I don't understand what you meant.  I was saying any changes to yolov5-cpu.env and yolov5-gpu.env files. but i am only seeing sh script file changes [on the file changes](https://github.com/intel-retail/automated-self-checkout/pull/415/files).

**Review by antoniomtz — APPROVED** [review]
LGTM

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
issue #410 saying yolov5-cpu.env and yolov5-gpu.env clean up but i dont' see any changes. this is confusing

**Comment by vli11** [comment]
> issue #410 saying yolov5-cpu.env and yolov5-gpu.env clean up but i dont' see any changes. this is confusing

I fixed #410 according to the description, sorry for the confusion, I've updated the issue title on #410.

**Review by brian-intel — APPROVED** [review]

**Pull request merged**